build.sbt

```diff
@@ -39,10 +39,13 @@ val backendDependencies = Seq(
   "com.lightbend.akka" %% "akka-stream-alpakka-csv" % alpakkaVersion,
 
   // IRI helper...
-  "org.apache.jena" % "jena-iri" % "3.9.0",
+  "org.apache.jena" % "jena-iri" % "3.17.0",
 
   // Ontology
-  "eu.ehri-project" % "ehri-definitions" % backendVersion
+  "eu.ehri-project" % "ehri-definitions" % backendVersion,
+
+  // Conversion to RDF
+  "com.herminiogarcia" % "shexml_2.12" % "0.3.1"
```

Comment:
Is this lib compatible with Scala 2.13?

Reply:
Not yet, I always wanted to enable the cross compilation but never really had the time to do it. If you plan to update I can try to enable it for 2.13. Nevertheless, for Scala 3 that's another story due to the dependencies that are still not updated.